src/ProfilesPage/components/UserInfo.js

```diff
@@ -23,7 +23,7 @@ const UserInfo = (props) => {
   if (props.userInfo) {
 
 
-    const { id, suspended, firstName, lastName, bio, block_connection_requests, admin } = props.userInfo;
+    const { id, suspended, firstName, lastName, bio, block_connection_requests, admin, superuser } = props.userInfo;
```

Comment:
I don't think you need this `superuser` here. Actually, based on what's used below, you should only need `firstName`, `lastName`, and `bio`.

Reply:
done